### DISCUSSION TITLE
Markdown shortcut for code blocks ```$lang followed by a new line?

### DISCUSSION THREAD

**Discussion by jessicarush** [mdx-editor/editor#533] [Ideas]
The current markdown shortcut for creating a code block is:

Type ` ```$lang  ` (with $lang being any supported language, followed by **space**).

I was actually stumped by this for longer than I care to admit, as I intuitively always hit the return key after typing ` ```$lang `.  My coworkers also do this so I don't think I'm alone. Is there a way to extend the `markdownShortcutPlugin` to also handle  creating code blocks with:

Type ` ```$lang ` (with $lang being any supported language, followed by a **new line**).

Is this possible?

**Comment by aleph2c** [comment]
I agree.  It's also hard to figure out why the code block doesn't work when you don't add the space at the end of the `$lang` part of the syntax.  This is not an intuitive behavior, it's in your docs, but it's to hard figure out what is going wrong with a google search phrase.  It just feels like a bug, even though it is documented.

**Comment by petyosi** [comment]
@jessicarush @aleph2c thank you for sharing that - I agree that this can be improved. Do you want to open a PR for a fix? The code is actually part of the markdown shortcuts defined by Lexical.
